ci/update-checker/src/main/scala/com/twitter/intellij/updatechecker/UpdateChecker.scala

```diff
@@ -0,0 +1,93 @@
+package com.twitter.intellij.updatechecker
+
+import java.nio.file.Path
+
+import org.jetbrains.intellij.pluginRepository._
+import org.jetbrains.intellij.pluginRepository.model.{PluginUpdateBean, ProductFamily}
+import org.virtuslab.ideprobe.Config
+import org.virtuslab.ideprobe.Extensions.PathExtension
+import org.virtuslab.ideprobe.dependencies.Plugin
+import org.virtuslab.ideprobe.dependencies.Plugin.Versioned
+
+import scala.jdk.CollectionConverters._
+
+object UpdateChecker {
+
+  // Assuming this program is launched in the project's base directory
+  private val configFile = Path.of("tests/src/test/resources/versions.conf")
+
+  case class PluginUpdate(id: String, version: String)
+
+  private val updateBeanOrdering = Ordering.fromLessThan[PluginUpdateBean] {
+    case (x, y) => x.getCdate < y.getCdate
+  }
+
+  private def updatedConfig(oldConfig: String, changes: Seq[PluginUpdate]): String = {
+    changes.foldLeft(oldConfig){ case (config, PluginUpdate(id, version)) =>
+      config.replaceFirst(
+        s"""\\{ id = "$id", version = "[a-zA-Z0-9_\\-.]+"""",
+        s"""{ id = "$id", version = "$version""""
+      )
+    }
+  }
+
+  def main(args: Array[String]): Unit = {
+
+    println(s"Reading the conf file from $configFile")
+    val config = Config.fromFile(configFile)
+    val intellijBuildVersion = config[String]("versions.intellij.build")
```

Comment:
create a case class, for example: `VersionsConf(intellij: IntelliJVersion, plugins: Map[String, Plugin])` and read it as `config[VersionsConf]("versions")`. Then modify it (changing plugins and intellij version and then serialize back Map("versions" -> versionsConf) (or create another case class to represent this main key). You can serialize it similarly as in release-tool, to json.

Reply:
I'll do that in the next pull request.